public/v2/apps/discourse.json

```diff
@@ -0,0 +1,121 @@
+{
+    "captainVersion": "2",
+    "documentation": "Taken from https://hub.docker.com/r/bitnami/discourse",
+    "displayName": "Discourse",
+    "description": "Discourse is the 100% open source discussion platform built for the next decade of the Internet. Use it as a mailing list, discussion forum, long-form chat room, and more!",
+    "dockerCompose": {
+        "version": "2",
+        "services": {
+            "$$cap_appname-postgresql": {
+                "documentation": "Taken from https://hub.docker.com/r/bitnami/postgresql",
+                "image": "docker.io/bitnami/postgresql:$$cap_postgresql_version",
+                "notExposeAsWebApp": "true",
+                "restart": "always",
+                "environment": ["ALLOW_EMPTY_PASSWORD=no"],
+                "volumes": ["$$cap_appname-postgresql-data:/bitnami/postgresql"]
+            },
+            "$$cap_appname-redis": {
+                "documentation": "Taken from https://hub.docker.com/r/bitnami/redis",
+                "image": "docker.io/bitnami/redis:$$cap_redis_version",
+                "notExposeAsWebApp": "true",
+                "restart": "always",
+                "environment": ["ALLOW_EMPTY_PASSWORD=no"],
+                "volumes": ["$$cap_appname-redis-data:/bitnami"]
+            },
+            "$$cap_appname": {
+                "documentation": "Taken from https://hub.docker.com/r/bitnami/discourse",
+                "image": "docker.io/bitnami/discourse:$$cap_discourse_version",
+                "containerHttpPort": "3000",
+                "depends_on": [
+                    "$$cap_appname-postgresql",
+                    "$$cap_appname-redis"
+                ],
+                "volumes": ["$$cap_appname-data:/bitnami"],
+                "environment": [
+                    "POSTGRESQL_HOST=$$cap_appname-postgresql",
```

Comment:
Change this to `srv-captain--$$cap_appname-postgresql`

Reply:
Okay, I'll try that now